sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/HPEMarketing.tsx

```diff
@@ -28,6 +33,8 @@ export const HPEMarketing = () => {
       <HPEVideo />
       <HPEPrivateCloud />
       <HPEProducts />
+      <HPEAIUseCase />
```

Comment:
This seems to be showing up twice on the page. Maybe now it should be remove if it's embedded in products?
```suggestion
```